### ISSUE TITLE
zlib not found

### ISSUE THREAD

**Issue by fabianfiorotto** [expresso-php/expresso-php#64]
I had to add `libzip-dev` to php/Dockerfile in order to be able to build the php docker image.
I'm using linux mint 18.3

```
RUN apt-get update && apt-get install -y libpng-dev libjpeg-dev libpq-dev libzip-dev \
    && rm -rf /var/lib/apt/lists/* \
    && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
    && docker-php-ext-install gd mbstring opcache pdo pdo_mysql pdo_pgsql zip mysqli
```

**Comment by Luukyb** [comment]
Hi @mtem27,
Looks like you included the fix for this with SOLR.
Can you confirm you had the same problem? 
Would this be fixed for both Apache anc Nginx?
Thanks,
Luc

**Comment by Luukyb** [comment]
Hi @fabianfiorotto ,
This is now fixed as part of Issue #28.
Thanks,
Luc
